internal/testutil/dict.go

```diff
@@ -15,15 +15,68 @@
 package testutil
 
 import (
+	"bytes"
 	"encoding/binary"
 	"fmt"
 	"math"
+	"os"
+	"testing"
+
+	"github.com/pebbe/dictzip"
```

Comment:
Get rid of the pebbe/dictzip dependency.

Currently `dictzip.Write` doesn't work on windows but `ianlewis/dictzip` doesn't support writing yet.